components/script/dom/window.rs

```diff
@@ -1019,26 +1019,33 @@ impl Window {
     /// and no reflow is performed. If reflow is suppressed, no reflow will be
     /// performed for ForDisplay goals.
     ///
-    /// TODO(pcwalton): Only wait for style recalc, since we have off-main-thread layout.
-    pub fn force_reflow(&self, goal: ReflowGoal, query_type: ReflowQueryType, reason: ReflowReason) {
+    /// TODO(pcwalton): Only wait for style recalc, since we have
+    /// off-main-thread layout.
+    ///
+    /// Returns true if the reflow was not suppressed, false otherwise.
```

Comment:
How about "returns true if the layout actually happened or false if it was suppressed"? The double negative is a bit confusing :)